src/wallet/sqlite.h

```diff
@@ -53,6 +53,8 @@ class SQLiteDatabase : public WalletDatabase
 
     const std::string m_file_path;
 
+    void Cleanup();
```

Comment:
70a2c658549c8a0caa3e58dcc7894d2bb8872cd5
```suggestion
    void Cleanup() noexcept;
```

Reply:
Done